client/reader/stream/index.jsx

```diff
@@ -60,10 +61,56 @@ function refreshCardFactory( post ) {
 
 const defaultCardFactory = config.isEnabled( 'reader/refresh/stream' ) ? refreshCardFactory : oldCardFactory;
 
+const MIN_DISTANCE_BETWEEN_RECS = 4; // page size is 7, so one in the middle of every page and one on page boundries, sometimes
+const MAX_DISTANCE_BETWEEN_RECS = 49; // page size is 7, so 7 pages
+const RECS_PER_BLOCK = 2;
+
+function getDistanceBetweenRecs() {
+	// the distance between recs changes based on how many subscriptions the user has.
+	// We cap it at MAX_DISTANCE_BETWEEN_RECS.
+	// It grows at the natural log of the number of subs, times a multiplier, offset by a constant.
+	// This lets the distance between recs grow quickly as you add subs early on, and slow down as you
+	// become a common user of the reader.
+	const totalSubs = FeedSubscriptionStore.getTotalSubscriptions();
```

Comment:
currently:
- from 1-9: is equals 4
- from 10-19 it increases 5-->8
- from 20-30 it increases from 9-->11
- ....
- from 400-600 it increases from 24-->26
- etc.

I don't feel super strong about this, but I'm not sure what the benefit to the user is of having this work so continuously as opposed to as a step function.  aka:
```javascript
if ( subs < 10 )
	return 4;
} else if ( subs < 20) {
	return 8;
else if ( subs < 50 ) {
	return 12;
} else if ( subs < 100) {
	return 20
} else if ( subs < 1000) {
	return 30;
}
```

The only reason I prefer a step function is that it is easier to reason about. If you think the UX of the continuous version is better, then lets stick with it 👍